### ISSUE TITLE
E-Mail address in reset password form is case-sensitive

### ISSUE THREAD

**Issue by subesokun** [RocketChat/Rocket.Chat#4158]
Hi,

we had the strange error that a user couldn't reset his password. It turned out that the user had a E-Mail address looking like "Some.User@company.com" but he used the E-Mail address "some.user@company.com" when trying to reset his password. 

Following error gets then logged:

```
I20160831-13:03:40.148(0) Exception while invoking method 'sendForgotPasswordEmail' Error: No such email for user
```

Is there an option to make the E-Mail validation case-insensitive?

Thanks a lot :)

Your Rocket.Chat version: 0.37.1

**Comment by graywolf336** [comment]
Was fixed via 7acbcfd848f835939600b69cc8facafc8af22643

**Comment by lpilz** [comment]
This bug is reoccurring in 3.2.2.
